docassemble/ALAffidavitOfIndigency/data/questions/affidavit_body.yml

```diff
@@ -58,6 +57,16 @@ code: |
 code: |
   is_indigent = public_assistance_kinds.any_true() or household_income_qualifies or not can_afford
 ---
+depends on:
+  - public_assistance_kinds
+  - household_income_qualifies
+  - can_afford
+code: |
+  need_supplement = not public_assistance_kinds.any_true() and not household_income_qualifies and not can_afford
+---
+code: |
+  need_income = not public_assistance_kinds.any_true()
```

Comment:
Similarly, this should get a `depends on` modifier (make sure that works--might need to use an `on change` block instead)

Reply:
Can't actually get it work with either. :/  I think it has something to do with the issues I mentioned in https://github.com/SuffolkLITLab/docassemble-ALAffidavitOfIndigency/pull/6#discussion_r982789574.